.github/workflows/ci-pytorch-test-full.yml

```diff
@@ -111,13 +111,15 @@ jobs:
       env:
         PACKAGE_NAME: ${{ matrix.pkg-name }}
       run: |
-        pip install -e . pytest --upgrade --find-links ${TORCH_URL}
+        pip install -e . --upgrade --find-links ${TORCH_URL}
         pip list
 
     - name: DocTests PL
-      if: ${{ matrix.pkg-name != 'LAI' }}
-      working-directory: src
-      run: python -m pytest pytorch_lightning
+      working-directory: ./src
+      run: |
+        pip install pytest -q
```

Comment:
Same question here

Reply:
Will mark this as resolved as it has the same question/answer, so we can keep just one sample :)